android/src/main/java/com/imagepicker/ImagePickerModule.java

```diff
@@ -308,6 +311,8 @@ public void launchImageLibrary()
   @ReactMethod
   public void launchImageLibrary(final ReadableMap options, final Callback callback)
   {
+    if (this.callback != null) return;
```

Comment:
what worries me slightly about this approach is that we can now have launchImageLibrary calls hang forever. In practice, it's probably fine though

Reply:
maybe we should call `responseHelper.invokeCancel(callback)`?

Reply:
I think every possible case is handled:
https://github.com/ExodusMovement/react-native-image-picker/blob/d630003291836be6cafa0e54a07d11f8722c599b/android/src/main/java/com/imagepicker/ImagePickerModule.java#L367
https://github.com/ExodusMovement/react-native-image-picker/blob/d630003291836be6cafa0e54a07d11f8722c599b/android/src/main/java/com/imagepicker/ImagePickerModule.java#L397
https://github.com/ExodusMovement/react-native-image-picker/blob/d630003291836be6cafa0e54a07d11f8722c599b/android/src/main/java/com/imagepicker/ImagePickerModule.java#L427
https://github.com/ExodusMovement/react-native-image-picker/blob/d630003291836be6cafa0e54a07d11f8722c599b/android/src/main/java/com/imagepicker/ImagePickerModule.java#L207